Python/1. print.md

```diff
@@ -59,7 +73,25 @@ print("Moi number is: " + 3)
 ### Output
 
 ```shell
-ERROR
+Traceback (most recent call last):
+  File "<stdin>", line 1, in <module>
+TypeError: can only concatenate str (not "int") to str
+```
+
+You can fix this by using commas or manually setting 3 to a string.
```

Comment:
the 'using commas' part is not needed. We already stated that it works with commas, and only with the + this problem occurs